cvpy/image/Image.py

```diff
@@ -352,6 +352,45 @@ def convert_wide_to_numpy(wide_image) -> np.ndarray:
         # Return the numpy array
         return np.frombuffer(wide_image[4 * 8:], dtype=np_data_type).reshape(height, width, num_channels)
 
+
+    @staticmethod
+    def convert_numpy_to_wide(numpy_array: np.ndarray) -> bytes:
+
+        """
+        Convert a numpy image array to a wide image.
+
+        Parameters
+        ----------
+        numpy_array: np.ndarray
+             Specifies the numpy image array
```

Comment:
Could you add a period to the end of this sentence. Otherwise, it looks great to me. Thanks for working on this, @courtneyambrozic!